jobs/multi-arch-pipeline.Jenkinsfile

```diff
@@ -285,13 +285,20 @@ stages:
 - id: ExecOrder 1 Stage
   execution_order: 1
   description: Stage 1 execution base
-  build_artifacts: [qemu]
-  require_artifacts: [ostree]
   prep_commands:
     - cat /cosa/coreos-assembler-git.json
+  require_artifacts: [ostree]
   post_commands:
+    - cosa buildextend-qemu
     - cosa kola run --basic-qemu-scenarios
+    - cosa kola run --parallel 4
```

Comment:
Do these two write/overwrite the same directory? Do I need to specify `--output-dir` here too?

Reply:
updated it so that they all specify `--output-dir`